### DISCUSSION TITLE
Keybaord shortcut to open table

### DISCUSSION THREAD

**Discussion by bombillazo** [dbeaver/dbeaver#21421] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
Similar to how we can user CMD + P to open files in VSCode, it would be great to be able to open up a table with a shortcut (ideally customizable).

**Describe the solution you'd like**
Add a key binding action that allows opening a table by promoting a table list for the currently focused schema (like when opening a new script opens on the focused schema)

**Describe alternatives you've considered**
We have to manually navigate the schema and table structure to find the desired table.

**Additional context**

The same concept could technically be extended to other items in the schema (datatypes, functions, etc)...

It would be like a search command palette

**Comment by E1izabeth** [comment, marked as answer]
Have you tried using this dialog with `Ctrl+Shift+D`
![image](https://github.com/dbeaver/dbeaver/assets/28875055/f6735f15-bc60-486c-a66b-585569e731ac)

**Comment by bombillazo** [comment]
Aaah thats it! I can customize the keyboard shortcut too! thank you!
